turbo.json

```diff
@@ -102,6 +102,27 @@
       "cache": false
     },
     "lint": {
+      "dependsOn": ["lint:oxlint"],
```

Comment:
the approach we took for vitest in `clerk-js` was to update the `lint` task to run both jest and vitest. not saying we _should_ do it that way here, just offering it in the event that sounds better than creating a new turbo task

Reply:
I kinda like that we can cache these individually and, I feel, that it enforces a generic interface (turbo) with which we run commands.